src/frontend/src/handler/create_secret.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2024 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use pgwire::pg_response::{PgResponse, StatementType};
+use prost::Message;
+use risingwave_sqlparser::ast::{CreateSecretStatement, SqlOption, Value};
+
+use crate::error::{ErrorCode, Result};
+use crate::handler::{HandlerArgs, RwPgResponse};
+use crate::{Binder, WithOptions};
+
+const SECRET_BACKEND_KEY: &str = "backend";
+
+const SECRET_BACKEND_META: &str = "meta";
+const SECRET_BACKEND_HASHICORP_VAULT: &str = "hashicorp_vault";
+
+pub async fn handle_create_secret(
+    handler_args: HandlerArgs,
+    stmt: CreateSecretStatement,
+) -> Result<RwPgResponse> {
+    let session = handler_args.session.clone();
+    let db_name = session.database();
+    let (schema_name, connection_name) =
+        Binder::resolve_schema_qualified_name(db_name, stmt.secret_name.clone())?;
```

Comment:
Do we follow the same rule of identifiers for secrets? That is, 

- `foo` will be converted to lower-case, so that `Foo` and `foo` will be the same
- `"foo"` will be interpreted as case-sensitive, i.e. `"foo"` and `"Foo"` are different

Reply:
For the secret name, yes. The impl leverages the existing funds and tries best to keep the same as other catalogs.